src/Compilers/CSharp/Test/Semantic/Semantics/FunctionPointerTests.cs

```diff
@@ -1563,7 +1667,6 @@ public static void M()
     }
 }");
 
-            // Some of these errors should become CS0306 after variant conversions are implemented, tracked by https://github.com/dotnet/roslyn/issues/39865
```

Comment:
None of the remaining 0411 errors can be inferred: they're between function pointers with int and string parameters. There is no reference conversion or implicit pointer conversion, so nothing can be inferred.